omod/src/main/webapp/resources/js/view/list.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+define(
+    [
+        openhmis.url.backboneBase + 'js/lib/backbone',
+        openhmis.url.backboneBase + 'js/lib/backbone-forms',
+        openhmis.url.backboneBase + 'js/lib/backbone.bootstrap-modal',
+        openhmis.url.backboneBase + 'js/view/list'
+    ],
+    function (Backbone) {
+
+        var Form = Backbone.Form,
+            editors = Form.editors;
+
+        /**
+         * LIST
+         *
+         * An array editor. Creates a list of other editor items.
+         *
+         * Special options:
+         * @param {String} [options.schema.itemType]          The editor type for each item in the list. Default: 'Text'
+         * @param {String} [options.schema.confirmDelete]     Text to display in a delete confirmation dialog. If falsey, will not ask for confirmation.
+         */
+        editors.InventoryList = editors.List.extend({

```

Comment:
Is this list specific to the inventory module? Couldn't the same logic be used in other lists that need some kind of confirmation when removing an item? It seems like this belongs in the backboneforms module so that it can easily be reused.